zuul.d/base.yaml

```diff
@@ -197,6 +197,25 @@
               tenant:
                 ip: 172.19.0.100
                 config_nm: false
+#
+# Base jobs using ci-bootstrap layout
+#
+- job:
+    name: cifmw-podified-multinode-edpm-ci-bootstrap-staging
+    parent: base-extracted-crc-ci-bootstrap-staging
+    timeout: 10800
+    attempts: 1
+    nodeset: *multinode_edpm_nodeset
+    irrelevant-files: *ir_files
+    required-projects: *multinode_edpm_rp
+    roles: *multinode_edpm_roles
+    pre-run:
+      - ci/playbooks/crc/reconfigure-kubelet.yml
```

Comment:
This thing was extracted from the bootstrap playbooks and nudged it here, isn't it? Just to confirm.

Reply:
yes correct, and this one doesn' t need to be on trusted repos.
added that as a comment in commit message also.